### PR TITLE
Medical HUDs no longer show viruses.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Medical HUDs no longer show viruses.

## Why It's Good For The Game

Viruses are really, really, really, really easy to spot and find by medbay. You don't need to check for symptoms before investigating, you don't need to figure out if that coughing is bad lungs or a virus, you just wait to see the funny hud icon that tells you all you need to know about a virus(how dangerous it is).

Frankly, it is to the point of not needing a virologist or any virology equipment at all for 99% of virus outbreaks, let alone sentient diseases.

Making doctors have to actually keep an eye out for symptoms should make viruses more interesting mechanically beyond "did i see the funny face icon or not".

## Changelog

:cl:
balance: Medical HUDs no longer show viruses.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
its the whole reason behind the stealth 2 threshold to hide from doctors

**Comment by GoldenAlpharex** [comment]
Yeah stealth isn't *that* hard to raise.

**Comment by D4C-420** [comment]
> Frankly, it is to the point of not needing a virologist or any virology equipment at all for 99% of virus outbreaks, let alone sentient diseases.

That has more to do with how its chems job to cure viruses and not virology's for some reason and less to do with spotting them. Also health scanners would still show your sick when scanning yourself ( baring stealth 2)

**Comment by Iamgoofball** [comment]
Stealth or not, I'm just not sold on the funny hud icon faces as a gameplay feature.

**Comment by GuillaumePrata** [comment]
A viroHUD glasses with this feature would help split virologists from doctors and stop the inevitable switch to surgeon/chemist that most virologist do after they speedrun a good disease.

Still this alone is a good change to make the virologist's job be more interesting IMO.

**Comment by the-orange-cow** [comment]
Would the AI/silicons still see viruses?

**Comment by zxaber** [comment]
Borgs use medical hud, so no.

Honestly the worst part of this PR is that viruses are already negative fun and this is just going compound that by ensuring maximum spread before anyone notices.

**Comment by moocowswag** [comment]
Doctors don`t go around the station looking for people who have a virus because people tend to self diagnose and flock medbay screaming "bad virus" as soon as a single symptom appears.

The main change this will have is doctors not seeing a red/black sad face next to someone and therefore not know the lethality of a virus until people start catching on fire which yes is the apparent goal of your pr to pay attention to patients and symptoms. 

But this could WILDLY increase the chances of a lethal virus actually spiraling into "one of those rounds" where most of the crew get wiped out which always ends in extreme amounts of salt from a large portion of the players.

This also compounds with the fact that lethal symptoms tend to pop up later than nonlethal ones so youll have people running up to medical staff and infecting them thinking its just some ordinary cough and then you`ll run into some potential salt between the players along with an expectation that medical staff need to be ready to turn on their internals at all times to avoid someone running up to them an infecting them.

**Comment by necromanceranne** [comment]
I feel like for this change to make sense, the Stealth stat on viruses probably needs to have some different effect. Possibly suppressing notifications and bubble text when an effect triggers based on the stealth stat in question. Maybe you get the 'effect' of a sneeze or cough but nobody is alerted every time it happens at especially high stealth.

**Comment by Livrah** [comment]
> Frankly, it is to the point of not needing a virologist or any virology equipment at all for 99% of virus outbreaks

1% of viro outbreak is more then enough.

**Comment by Watermelon914** [comment]
if a virus doesn't have any stealth it should be diagnosable with the medical HUD

**Comment by Shadowflame909** [comment]
I was previously unconcerned by this PR. But I just played a round with a high stealth level virus. The virus wasn't lethal, but it gave you organ damage, coughing, and Confusion. Which meant that my key inputs were useless and I was at the mercy of screaming for help to get dragged to medbay.

It was very horrible.

Virology is not fun. Please do not make it harder to fight, and have the crew rely on one person who is most likely the one responsible for the torturous contraptions. (Who else but virology mains can make viruses as annoying as that one?)

**Comment by moocowswag** [comment]
(Who else but virology mains can make viruses as annoying as that one?)

Back during miasma changes we had a pretty large sample amount of viruses that could be generated from gibs, a lot of them didnt have much stealth but a fair amount of them did have debilitating or minorly lethal effects.

**Comment by GuillaumePrata** [comment]
Alternatively, we use this as an excuse to start a rebalance on how effective the symptoms are.
Right now everything needs to be powerful and fast because diseases are so easy to see from a mile away. Any Doctor or Silicon that randomly walks by can see a disease and it takes zero effort on their part.

**Comment by thatoneplebeian** [comment]
While I do believe that Virology is repetitive and in need of a rework (and that you made this PR because you feel similarly) I am not convinced you have a good feel for how this will actually affect a round, or how these systems play out as they are now. 

I do not believe this change will actually accomplish your goal of making doctors investigate diseases more thoroughly and use the Virology equipment to check infected players' symptoms. 

As others have stated, were this merged, it would only really affect player-created viruses. This is because as soon as someone starts experiencing any kind of symptoms they will immediately rush to medbay to get diagnosed and cured. Even without the medical HUD displaying a "funny face", any doctor scanning them and seeing an unknown disease would just ask the virologist over radio if they had released a disease. If the answer was no, the cure could be made without checking the symptoms. 
This happens already, when a sentient disease has infected a player with no symptoms or with only beneficial symptoms. Despite the blue funny face, any experienced player will just check in with the viro and cure it.

When it comes to player-created diseases, what I believe this change will do is punish players more for trusting the virologist, and so it will increase distrust of the virologist. Because the symptoms cannot be checked without using the virologist's machine, players will have to simply trust that a distributed virus is safe. Sure, the CMO can check it first but this is something that already happens in game, and it is easy for the viro to just swap out diseases. The virologist can also stealthily infect unsuspecting players. With this change in place, whether or not the virus is good or bad will only be clear once bad symptoms have already started manifesting and disabling/killing players. At that point there will be a mad rush to mix the cure and make a vaccine...which again, already happens without this change.

Basically, this change will not actually increase player interaction with virology systems. Instead it
- Makes player-created diseases more deadly
- Increases the likelihood of the players lynching the virologist (as moocowswag pointed out)

I also would like to see more interaction with virology but I don't believe this change will move the game toward that goal.

**Comment by Mothblocks** [comment]
This is the point of the stealth stat, which I see with the purpose of:

A. Make the overall virus worse, since the things that provide good stealth tank/don't contribute to other stats, and take the place of a deadlier symptom.
B. Separate viro diseases (handcrafted by players, the *exact one* thing a virologist antagonist can do) from random ones

This is a massive virology buff when dying to a virus already is not fun

**Comment by Mothblocks** [comment]
You also didn't even remove them from the dmi :(
